### ISSUE TITLE
Video chat Black screen (Jitsi Meet)

### ISSUE THREAD

**Issue by franckadil** [RocketChat/Rocket.Chat.Android#2326]
## Describe the bug

A black screen is displayed when starting or joining the video chat. And nothing happens

## To Reproduce
1. Go to video chat
2. Lauch or click join a videochat 

## Expected behavior
Start the video chat

## Logs
```
2019-05-19 04:15:03.841 23661-23938/chat.rocket.android.dev D/ReactNativeJS: '[modules/RTC/RTCUtils.js]', '<>: ', 'Available devices: ', [ { kind: 'videoinput', label: '0', groupId: '', deviceId: '0' },
      { kind: 'videoinput', label: '1', groupId: '', deviceId: '1' },
      { kind: 'audioinput',
        label: 'Audio',
        groupId: '',
        deviceId: 'audio-1' } ]
2019-05-19 04:15:03.959 23661-23938/chat.rocket.android.dev I/ReactNativeJS: '[__filename]', '<>: ', 'Extending config with: {"startAudioOnly":false,"startWithAudioMuted":true,"startWithVideoMuted":true}'
2019-05-19 04:15:03.968 23661-23938/chat.rocket.android.dev I/ReactNativeJS: '[modules/RTC/ScreenObtainer.js]', '<>: ', 'Screen sharing not supported by the current browser: ', 'react-native'
2019-05-19 04:15:03.969 23661-23938/chat.rocket.android.dev I/ReactNativeJS: '[modules/RTC/ScreenObtainer.js]', '<>: ', 'Desktop sharing disabled'
2019-05-19 04:15:03.970 23661-23938/chat.rocket.android.dev I/ReactNativeJS: '[__filename]', '<>: ', 'Start muted: '
2019-05-19 04:15:03.973 23661-23938/chat.rocket.android.dev I/ReactNativeJS: '[__filename]', '<>: ', 'Start audio only set to false'
2019-05-19 04:15:03.973 23661-23938/chat.rocket.android.dev I/ReactNativeJS: '[__filename]', '<>: ', 'Audio-only disabled'
2019-05-19 04:15:03.987 23661-23938/chat.rocket.android.dev W/ReactNativeJS: '[__filename]', '<>: ', 'Calendar access not granted.'
2019-05-19 04:15:03.988 23661-23938/chat.rocket.android.dev D/ReactNativeJS: '[modules/RTC/RTCUtils.js]', '<>: ', 'Available devices: ', [ { kind: 'videoinput', label: '0', groupId: '', deviceId: '0' },
      { kind: 'videoinput', label: '1', groupId: '', deviceId: '1' },
      { kind: 'audioinput',
        label: 'Audio',
        groupId: '',
        deviceId: 'audio-1' } ]
2019-05-19 04:15:03.998 23661-23938/chat.rocket.android.dev W/ReactNativeJS: '[__filename]', '<>: ', 'Error creating analytics handler: TypeError: undefined is not an object (evaluating \'m.parentNode\')'
2019-05-19 04:15:03.998 23661-23938/chat.rocket.android.dev D/ReactNativeJS: '[__filename]', '<>: ', 'Loaded 1 analytics handlers'
2019-05-19 04:15:04.008 23661-23701/chat.rocket.android.dev D/EGL_emulation: eglMakeCurrent: 0xc18fdb60: ver 3 0 (tinfo 0xc1937370)
2019-05-19 04:15:04.148 23661-23938/chat.rocket.android.dev W/ReactNativeJS: '[__filename]', '<>: ', 'Error creating analytics handler: TypeError: undefined is not an object (evaluating \'m.parentNode\')'
2019-05-19 04:15:04.149 23661-23938/chat.rocket.android.dev W/ReactNativeJS: '[__filename]', '<>: ', 'Error creating analytics handler: TypeError: undefined is not an object (evaluating \'m.parentNode\')'
2019-05-19 04:15:04.149 23661-23938/chat.rocket.android.dev D/ReactNativeJS: '[__filename]', '<>: ', 'Loaded 1 analytics handlers'
2019-05-19 04:15:06.063 1659-1743/? W/audio_hw_generic: Not supplying enough data to HAL, expected position 9866224 , only wrote 9642007
```
## Logs 2

```
2019-05-19 04:18:12.979 23661-23661/chat.rocket.android.dev D/ChatRoomAdapter: index: 0
2019-05-19 04:18:13.156 23661-24292/chat.rocket.android.dev D/AppModule$provideHttpLoggingInterceptor$interceptor: --> POST https://[redacted].junglewp.com/api/v1/video-conference/jitsi.update-timeout
2019-05-19 04:18:13.157 23661-24292/chat.rocket.android.dev D/AppModule$provideHttpLoggingInterceptor$interceptor: Content-Type: application/x-www-form-urlencoded
2019-05-19 04:18:13.157 23661-24292/chat.rocket.android.dev D/AppModule$provideHttpLoggingInterceptor$interceptor: Content-Length: 41
2019-05-19 04:18:13.157 23661-24292/chat.rocket.android.dev D/AppModule$provideHttpLoggingInterceptor$interceptor: User-Agent: RC Mobile; Android 10; v3.4.0 (2065)
2019-05-19 04:18:13.157 23661-24292/chat.rocket.android.dev D/AppModule$provideHttpLoggingInterceptor$interceptor: X-Auth-Token: EzNABe8PKAHP7rONIINoKgoZe0KPD20XTaHlpKr71qQ
2019-05-19 04:18:13.158 23661-24292/chat.rocket.android.dev D/AppModule$provideHttpLoggingInterceptor$interceptor: X-User-Id: jdZodaFoWrRjPZmeC
2019-05-19 04:18:13.158 23661-24292/chat.rocket.android.dev D/AppModule$provideHttpLoggingInterceptor$interceptor: roomId=jdZodaFoWrRjPZmeCvYzwqzkXSBpodzSmE
2019-05-19 04:18:13.158 23661-24292/chat.rocket.android.dev D/AppModule$provideHttpLoggingInterceptor$interceptor: --> END POST (41-byte body)
2019-05-19 04:18:13.324 23661-24292/chat.rocket.android.dev D/AppModule$provideHttpLoggingInterceptor$interceptor: <-- 200 OK https://[redacted].com/api/v1/video-conference/jitsi.update-timeout (164ms)
2019-05-19 04:18:13.324 23661-24292/chat.rocket.android.dev D/AppModule$provideHttpLoggingInterceptor$interceptor: Server: nginx/1.15.12
2019-05-19 04:18:13.324 23661-24292/chat.rocket.android.dev D/AppModule$provideHttpLoggingInterceptor$interceptor: Date: Sun, 19 May 2019 03:18:18 GMT
2019-05-19 04:18:13.324 23661-24292/chat.rocket.android.dev D/AppModule$provideHttpLoggingInterceptor$interceptor: Content-Type: application/json
2019-05-19 04:18:13.324 23661-24292/chat.rocket.android.dev D/AppModule$provideHttpLoggingInterceptor$interceptor: Transfer-Encoding: chunked
2019-05-19 04:18:13.325 23661-24292/chat.rocket.android.dev D/AppModule$provideHttpLoggingInterceptor$interceptor: Connection: keep-alive
2019-05-19 04:18:13.325 23661-24292/chat.rocket.android.dev D/AppModule$provideHttpLoggingInterceptor$interceptor: Access-Control-Allow-Origin: *
2019-05-19 04:18:13.325 23661-24292/chat.rocket.android.dev D/AppModule$provideHttpLoggingInterceptor$interceptor: X-Instance-ID: uhopDo4WMH5GDHKRF
2019-05-19 04:18:13.326 23661-24292/chat.rocket.android.dev D/AppModule$provideHttpLoggingInterceptor$interceptor: Cache-Control: no-store
2019-05-19 04:18:13.326 23661-24292/chat.rocket.android.dev D/AppModule$provideHttpLoggingInterceptor$interceptor: Pragma: no-cache
2019-05-19 04:18:13.326 23661-24292/chat.rocket.android.dev D/AppModule$provideHttpLoggingInterceptor$interceptor: Vary: Accept-Encoding
2019-05-19 04:18:13.328 23661-24292/chat.rocket.android.dev D/AppModule$provideHttpLoggingInterceptor$interceptor: {"jitsiTimeout":"2019-05-19T03:18:47.318Z","success":true}
2019-05-19 04:18:13.329 23661-24292/chat.rocket.android.dev D/AppModule$provideHttpLoggingInterceptor$interceptor: <-- END HTTP (58-byte body)
2019-05-19 04:18:21.680 23661-23797/chat.rocket.android.dev D/ConnectionManager$connect$$inlined$createBatchActor: Processing batch: 1
2019-05-19 04:18:21.680 23661-23797/chat.rocket.android.dev D/ConnectionManager$connect$$inlined$createBatchActor: Processing Users batch: 1 - 15
2019-05-19 04:18:21.681 23661-23805/chat.rocket.android.dev D/DatabaseManager$processUsersBatch: Converted users batch(1) in 0 MS
2019-05-19 04:18:21.682 23661-23805/chat.rocket.android.dev D/DatabaseManager: writerChannel: LinkedListChannel@1d063f1{ReceiveQueued}, closedForSend: false, closedForReceive: false, empty: true, full: false
2019-05-19 04:18:21.683 23661-23795/chat.rocket.android.dev D/DatabaseManager$doOperation: Upserted users batch(1) in 1 MS
2019-05-19 04:18:23.160 23661-24292/chat.rocket.android.dev D/AppModule$provideHttpLoggingInterceptor$interceptor: --> POST https://[redacted].com/api/v1/video-conference/jitsi.update-timeout
2019-05-19 04:18:23.161 23661-24292/chat.rocket.android.dev D/AppModule$provideHttpLoggingInterceptor$interceptor: Content-Type: application/x-www-form-urlencoded
2019-05-19 04:18:23.161 23661-24292/chat.rocket.android.dev D/AppModule$provideHttpLoggingInterceptor$interceptor: Content-Length: 41
2019-05-19 04:18:23.161 23661-24292/chat.rocket.android.dev D/AppModule$provideHttpLoggingInterceptor$interceptor: User-Agent: RC Mobile; Android 10; v3.4.0 (2065)
2019-05-19 04:18:23.161 23661-24292/chat.rocket.android.dev D/AppModule$provideHttpLoggingInterceptor$interceptor: X-Auth-Token: EzNABe8PKAHP7rONIINoKgoZe0KPD20XTaHlpKr71qQ
2019-05-19 04:18:23.162 23661-24292/chat.rocket.android.dev D/AppModule$provideHttpLoggingInterceptor$interceptor: X-User-Id: jdZodaFoWrRjPZmeC
2019-05-19 04:18:23.162 23661-24292/chat.rocket.android.dev D/AppModule$provideHttpLoggingInterceptor$interceptor: roomId=jdZodaFoWrRjPZmeCvYzwqzkXSBpodzSmE
2019-05-19 04:18:23.162 23661-24292/chat.rocket.android.dev D/AppModule$provideHttpLoggingInterceptor$interceptor: --> END POST (41-byte body)
2019-05-19 04:18:23.327 23661-24292/chat.rocket.android.dev D/AppModule$provideHttpLoggingInterceptor$interceptor: <-- 200 OK https://[redacted]/api/v1/video-conference/jitsi.update-timeout (164ms)
2019-05-19 04:18:23.328 23661-24292/chat.rocket.android.dev D/AppModule$provideHttpLoggingInterceptor$interceptor: Server: nginx/1.15.12
2019-05-19 04:18:23.328 23661-24292/chat.rocket.android.dev D/AppModule$provideHttpLoggingInterceptor$interceptor: Date: Sun, 19 May 2019 03:18:28 GMT
2019-05-19 04:18:23.328 23661-24292/chat.rocket.android.dev D/AppModule$provideHttpLoggingInterceptor$interceptor: Content-Type: application/json
2019-05-19 04:18:23.328 23661-24292/chat.rocket.android.dev D/AppModule$provideHttpLoggingInterceptor$interceptor: Transfer-Encoding: chunked
2019-05-19 04:18:23.328 23661-24292/chat.rocket.android.dev D/AppModule$provideHttpLoggingInterceptor$interceptor: Connection: keep-alive
2019-05-19 04:18:23.328 23661-24292/chat.rocket.android.dev D/AppModule$provideHttpLoggingInterceptor$interceptor: Access-Control-Allow-Origin: *
2019-05-19 04:18:23.328 23661-24292/chat.rocket.android.dev D/AppModule$provideHttpLoggingInterceptor$interceptor: X-Instance-ID: uhopDo4WMH5GDHKRF
2019-05-19 04:18:23.329 23661-24292/chat.rocket.android.dev D/AppModule$provideHttpLoggingInterceptor$interceptor: Cache-Control: no-store
2019-05-19 04:18:23.329 23661-24292/chat.rocket.android.dev D/AppModule$provideHttpLoggingInterceptor$interceptor: Pragma: no-cache
2019-05-19 04:18:23.329 23661-24292/chat.rocket.android.dev D/AppModule$provideHttpLoggingInterceptor$interceptor: Vary: Accept-Encoding
2019-05-19 04:18:23.330 23661-24292/chat.rocket.android.dev D/AppModule$provideHttpLoggingInterceptor$interceptor: {"jitsiTimeout":"2019-05-19T03:18:47.318Z","success":true}
2019-05-19 04:18:23.330 23661-24292/chat.rocket.android.dev D/AppModule$provideHttpLoggingInterceptor$interceptor: <-- END HTTP (58-byte body)
2019-05-19 04:18:32.382 23661-24356/chat.rocket.android.dev D/ConnectionManager$connect: GOT Room streamed
2019-05-19 04:18:32.886 23661-23798/chat.rocket.android.dev D/ConnectionManager$connect$$inlined$createBatchActor: Processing batch: 1
2019-05-19 04:18:32.886 23661-23799/chat.rocket.android.dev D/ConnectionManager$connect$$inlined$createBatchActor: Processing batch: 1
```

## Screenshots
![Screenshot_1558235222](https://user-images.githubusercontent.com/42721509/57977259-71343380-79ec-11e9-96f7-2bfd1fca291f.png)

## Devices and Versions

Your Rocket.Chat.Android version: 3.3.0
Your Rocket.Chat Server version: 1.0.2 / 1.0.3
Tested on Several devices ranging from Android 6.0 to 9.0

**Comment by ghalong** [comment]
I also have the same problem, and print more log 
`2019-06-06 13:31:04.843 31722-542/chat.rocket.android.dev E/com.amplitude.api.AmplitudeClient: Argument context cannot be null in initialize()
2019-06-06 13:31:04.848 31722-542/chat.rocket.android.dev E/com.amplitude.api.AmplitudeClient: context cannot be null, set context with initialize() before calling setUserProperties`

**Comment by franckadil** [comment]
@sampaiodiego any thoughts about this ?

**Comment by mperica** [comment]
The same bug for me also, only black screen

**Comment by MaybeSomeone** [comment]
same problem to me!! any suggesstions??

**Comment by philipbrito** [comment]
Issue was fixed guys! You can download the 3.5.0 beta version on Google Play bringing that fix already.
